### PR TITLE
Feat/atom label font size customization

### PR DESCRIPTION
**Pull request by tomasmax**
**What does this PR do?**
Add fontSize prop in order to be able to customize AtomLabel font size.

**Demo link**
Waiting to be generated...

**Demo Screenshot**
image

### CODE REVIEW

**Review by carlosvillu — DISMISSED** [review]

File: components/atom/label/src/index.js [hunk 1]
```diff
@@ -27,6 +35,10 @@ const AtomLabel = ({name, inline, text, optionalText, type, onClick}) => (
 
 AtomLabel.displayName = 'AtomLabel'
 
+AtomLabel.defaultProps = {
+  fontSize: FONT_SIZES.BASE
```
Comment (igncalleja) [inline components/atom/label/src/index.js]: I see no BASE in FONT_SIZES... ¿?

File: components/atom/label/src/index.scss [hunk 2]
```diff
@@ -6,14 +6,29 @@ $c-atom-label-type: success $c-success, error $c-error;
 
 .sui-AtomLabel {
   display: block;
-  font-size: $fz-base;
```
Comment (igncalleja) [inline components/atom/label/src/index.scss]: Shouldn't the label have a default font-size?

**Review by arnau-rius — DISMISSED** [review]

**Review by davecarter — DISMISSED** [review]

File: components/atom/label/src/index.scss (hunk 2 above)
Comment (tomasmax) [reply components/atom/label/src/index.scss]:
```
&--medium {
    font-size: $fz-base;
  }
``` 
is the default one

Comment (igncalleja) [reply components/atom/label/src/index.scss]: IMHO Then it is not a "modifier", but the default font-size and it should be in the main definition

**Review by carlosvillu — DISMISSED** [review]

**Review by davecarter — DISMISSED** [review]

**Review by davecarter — APPROVED** [review]

**Review by carlosvillu — APPROVED** [review]

**Pull request merged**